pypeit/calibrations.py

```diff
@@ -334,15 +349,17 @@ def get_align(self):
             self.alignments.is_synced(self.slits)
             return self.alignments
 
-        msalign = buildimage.buildimage_fromlist(self.spectrograph, self.det, self.par['alignframe'], align_files,
+        msalign = buildimage.buildimage_fromlist(self.spectrograph, self.det,
+                                                 self.par['alignframe'], align_files,
                                                  bias=self.msbias, bpm=self.msbpm)
 
         # Extract some header info needed by the algorithm
         binning = self.spectrograph.get_meta_value(align_files[0], 'binning')
 
         # Instantiate
-        alignment = alignframe.TraceAlignment(msalign, self.slits, self.spectrograph, self.par['alignment'],
-                                              det=self.det, binning=binning, qa_path=self.qa_path, msbpm=self.msbpm)
+        alignment = alignframe.TraceAlignment(msalign, self.slits, self.spectrograph,
```

Comment:
Do we need to pass the msbpm explicitly here? The msalign object has a mask attributed, and from that mask you know which pixels are bad. Is there a reason why you explicitly need the BPM?

Reply:
@rcooke-ast , any thoughts on this?  BPM was being passed in the previous version of the code.  I'm happy to change to whatever you think should be done.